lib/public/components/Log/logLinkButton.js

```diff
@@ -20,12 +20,10 @@ import { clipboardTarget } from '/components/common/clipboardTarget.js';
  * @param {Log} log the log object that this button corresponds to
  * @return {Component} The link button.
  */
-export const linkButton = (log) => {
+export const logLinkButton = (log) => {
+    // TODO: absolute url
```

Comment:
Please do this todo in this PR
You can extract a function `getFrontUrl` from `frontLink` component to do so
Feel free to ask me if you need any help

Reply:
Ah of course, thanks for catching that!